crates/gooey-kludgine/src/lib.rs

```diff
@@ -385,6 +385,30 @@ where
             );
         }
     }
+
+    /// A keyboard event occurred while the window was focused.
+    #[allow(unused_variables)]
```

Comment:
This shouldn't be here. I only put that on the trait definition to prevent needing `_` on parameter names. When I use Rust analyzer to implement a trait function, it doesn't bring in the attributes by default, and the parameter names are ready to be used rather than `_` prefixed.